### ISSUE TITLE
Heading component without attributes renders as subtitle

### ISSUE THREAD

**Issue by verheyenkoen** [couds/react-bulma-components#35]
This code:

```
<Heading>Test</Heading>
```

... renders this HTML:

```
<h1 class="title subtitle">Test</h1>
```

... which basically renders a subtitle. I guess this should not render the `subtitle` class.

**Comment by threeid** [comment]
Yeah, here is the link https://codesandbox.io/s/84vnzz1vo8
<img width="1224" alt="screen shot 2018-05-27 at 9 57 21 am" src="https://user-images.githubusercontent.com/37827220/40582078-7251d9e6-6194-11e8-9744-5e835a1f5c17.png">

It is weird! `subtitle: false` in `defaultProps`

**Comment by verheyenkoen** [comment]
Turns out there already was a PR for this: #31

**Comment by couds** [comment]
Hello, Sorry forr the delay, as @verheyenkoen said this was fixed on #31. 

I'm doing a release now with this a other 2 fixes.

Thanks for using this library
